athenet/algorithm/simple_neuron_deleter.py

```diff
@@ -0,0 +1,63 @@
+"""
+    sender - SimplE Neuron DEleteR
+
+    Deletes the least significant neurons. Value of neuron is defined as
+    sum of absolute values of weights outgoing from neuron divided by
+    sum of absolute values of weights outgoing from entire layer.
+"""
+
+import numpy as np
+
+from athenet.layers import FullyConnectedLayer
+from athenet.algorithm.utils import list_of_percentage_rows, delete_row
+
+
+def simple_neuron_deleter(network, config):
+    """
+        :network: - an instance of athenet.Network.
+        :config: - tuple of 2 foats, p and layer_limit
+        :p, layer_limit: - floats between 0 and 1.
+
+        Modifies [network]. Deletes [p] neurons from layers connected direclty
+        to fully connected layer's. Do not delete more than [layer_limit]
+        neurons from single layer.
+        If [layer_limit] < [p] then at most [layer_limit] neurons will be
+        deleted.
+
+        Deletion of neuron is simulated by setting all weights outgoing
```

Comment:
Ok.Changed: "Deletion of neuron is simulated by replacing all weights outgoing form to it by 0. "